apps/web/src/components/costs/average-costs-calculation.tsx

```diff
@@ -0,0 +1,117 @@
+interface EnergyData {
+    timestamp: string;
+    value: number;
+}
+
+interface UserData {
+    basePrice: number;
+    workingPrice: number;
+}
+
+const formatDate = (date: Date): string => date.toISOString().split("T")[0];
+
+export function calculateAverageCostsPerDay(energyData: EnergyData[], userData: UserData[]): number {
+    if (
+        !energyData ||
+        energyData.length === 0 ||
+        !userData ||
+        userData.length < 2 ||
```

Comment:
Warum < 2?

Reply:
Weil du von einem Reading nicht den Average berechnen kannst😄 kann ich aber auch weglassen

Reply:
Doch, kann man schon. Ist dann halt nur der eine Wert von dem Reading.
Deswegen macht es für mich jetzt nicht so viel Sinn, dass in dem Fall dann 0 zurückgegeben wird :smile: